docs/api/v2/node.md

```diff
@@ -88,6 +88,21 @@ Call signatures:
 
 ```Nim
 proc waku_version(): string
+<<<<<<< HEAD
```

Comment:
rebaser

Reply:
@oskarth you were looking at an old changeset